easybuild/easyconfigs/j/Jellyfish/Jellyfish-2.3.0-foss-2019a.eb

```diff
@@ -0,0 +1,40 @@
+##
+# This is a contribution from DeepThought HPC Service, Flinders University, Adelaide, Australia 
+# Homepage:     https://staff.flinders.edu.au/research/deep-thought
+#
+# Authors::     Robert Qiao <rob.qiao@flinders.edu.au> 
+# License::     Custom
+#
+# Notes::
+##
+
+easyblock = 'ConfigureMake'
+
+name = 'Jellyfish'
+version = '2.3.0'
+
+homepage = 'http://www.genome.umd.edu/jellyfish.html'
+description = "Jellyfish is a tool for fast, memory-efficient counting of k-mers in DNA."
+
+toolchain = {'name': 'foss', 'version': '2019a'}
```

Comment:
@robqiao I think you can use the `GCC` sub-toolchain instead? I don't immediately see any usage of BLAS or MPI in the sources